### ISSUE TITLE
disable case transform on login

### ISSUE THREAD

**Issue by emyoulation** [gramps-project/Gramps.js#237]
The login form field does a capitalization (Camel/Proper/Title case) text transform during entry of a username. Please disable this.

It is unclear if the username is case sensitive but but login data should not be transformed since part of the form is typed in the blind.

It also makes the user uncertain if the password field similarly has a case transform... and passwords almost always are case sensitive with a 2 case requirement.

**Comment by DavidMStraub** [comment]
I have a guess... :apple: 

![grafik](https://github.com/gramps-project/Gramps.js/assets/10965193/f35747ca-6213-4091-ae99-205c5bd28e5e)

**Comment by emyoulation** [comment]
Evidently, it affects Chrome on mobile Andriod occasionally too. Although Safari on iOS is apparently a frequent offender.



Found this on stack overflow:

[Bryce Howitson](https://stackoverflow.com/users/6268914/bryce-howitson)
https://stackoverflow.com/questions/55053718/prevent-ios-safari-from-capitalizing-the-first-letter-of-inputs#

As I was typing this a friend [pointed](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/autocapitalize) me to `HTMLInputElement Autocapitalize="off"` and that seems to do the trick very easily.

Turns out that this can be included with `autocomplete="off"` to mostly control how mobile browsers deal with inputs.

So basically 
`<input type="text" autocapitalize="off" required placeholder="username" /> `
solves my specific problem.

According to [iOS](https://developer.apple.com/library/archive/documentation/AppleApplications/Reference/SafariHTMLRef/Articles/Attributes.html#//apple_ref/doc/uid/TP40008058-autocapitalize) and [Android](https://developers.google.com/web/updates/2015/04/autocapitalize) docs this doesn't always work but it should solve the majority of problems.

Bonus: This isn't an all or nothing setting like allowing zoom. You can apply it selectively to inputs where it's needed and allow inputs that need more of a "copy" style content to continue helping the user.
